wesnoth-multiplayer.po

```diff
@@ -2098,22 +2112,16 @@ msgstr "4p — Isarova křižovatka"
 
 #. [multiplayer]: id=multiplayer_Isars_Cross
 #: data/multiplayer/scenarios/4p_Isars_Cross.cfg:9
-#, fuzzy
-#| msgid ""
-#| "A very small 2 vs. 2 map. Close quarters fighting means the successful "
-#| "rotation of units and planning of moves is important. Teams set for "
-#| "players 1&4 vs. 2&3 (northeast vs. southwest). Designed for 75 starting "
-#| "gold, 20 villages."
 msgid ""
 "A very small 2 vs. 2 map. Close quarters fighting means the successful "
 "rotation of units and planning of moves is important. Teams set for players "
 "1 &amp; 4 vs. 2 &amp; 3 (northeast vs. southwest). Designed for 75 starting "
 "gold, 20 villages."
 msgstr ""
-"Velmi malá mapa 2 proti 2. Pro boje ve stísněných podmínkách je klíčem k "
-"vítězství úspěšná rotace jednotek a dopředné plánování tahů. Doporučené "
-"rozdělení je 1&4 proti 2&3 (severovýchod proti jihozápadu). Navrženo pro 75 "
-"zlatých na začátku, obsahuje 20 vesnic."
+"Velmi malá mapa pro hru 2 proti 2. Pro boje ve stísněných podmínkách je "
+"klíčem k vítězství úspěšná rotace jednotek a dopředné plánování tahů. "
+"Doporučené rozdělení je 1&4 proti 2&3 (severovýchod proti jihozápadu). "
```

Comment:
\&amp; bude lepší, mezera je můj vklad
```suggestion
"Doporučené rozdělení je 1 &amp; 4 proti 2 &amp; 3 (severovýchod proti jihozápadu). "
```

Reply:
Jakože samotné & by nemuselo být korektně rozeznáno?

Reply:
Pango píše do konzole varování a to není hezké:

20201112 17:16:10 warning gui/layout: pango_text::validate_markup text 'Velmi malá mapa 2 proti 2. Pro boje ve stísněných podmínkách je klíčem k vítězství úspěšná rotace jednotek a dopředné plánování tahů. Doporučené rozdělení je 1&4 proti 2&3 (severovýchod proti jihozápadu). Navrženo pro 75 zlatých na začátku, obsahuje 20 vesnic.' has unescaped ampersands '&', escaped them.